marginaleffects/plot_common.py

```diff
@@ -30,15 +30,15 @@ def dt_on_condition(model, condition):
             key in modeldata.columns for key in condition_new.keys()
         ), "All keys of condition must be columns of the model."
         first_key = next(iter(condition_new))
-        to_datagrid = condition_new
+        to_datagrid = condition_new  # third pointer to the same object? looks like a BUG
```

Comment:
Do you have a proposed alternative to this?

Reply:
My guess is the original author wanted to modify `condition_new` without overwriting, because the original `condition_new` is used later.